bundle/Command/MultiprocessCommand/Items.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\SiteBundle\Command\MultiprocessCommand;
+
+class Items
```

Comment:
@pspanja a minor issue, but should this class be renamed? `Items $items, $items->getItems()` is a little confusing.

Reply:
It could be named `ItemList`

Reply:
Updated in https://github.com/netgen/site-bundle/pull/30/commits/c6b67c2d2367bea0426946783e11f0e5e3128a08

Reply:
:+1: